adcom1/content.go

```diff
@@ -207,6 +217,24 @@ type Content struct {
 	//   Refer to Object: Producer.
 	Producer *Producer `json:"producer,omitempty"`
 
+	// Attribute:
+	//   network
+	// Type:
+	//   object
+	// Definition:
+	//   Details about the network.
+	//   Refer to Object: Network.
+	Network *Network `json:"network,omitempty"`
+
+	// Attribute:
+	//   channel
+	// Type:
+	//   object
+	// Definition:
+	//   Details about the channel.
+	//   Refer to Object: Channel.
+	Channel *Channel `json:"channel,omitempty"`
```

Comment:
Just realized that I'm reviewing `adcom` changes though the scope of this was OpenRTB 2.6... Weird.

Reply:
> Right... The 1.0 FINAL doesn't mean that additions are added as 1.1 and so on. They are added to existing 1.0 FINAL document:

I was surprised by this too. I expected AdCom 1.1 or some such.

Reply:
So, any change to either the major or minor version is considered a breaking change. They don't follow semver. The versioning strategy is document at the end of the spec. I provided feedback that it throws off the semver expectation. The IAB isn't going to change this strategy now, but will consider a change of approach in 2.x.